assets/schema_input.json

```diff
@@ -19,6 +19,48 @@
                 "exists": true,
                 "pattern": "^\\S+\\.(fasta|fa|fna|ffn)(\\.gz)?$",
                 "errorMessage": "The assembly needs to be a fasta file, either '.fasta', '.fa', '.fna', or '.ffn' (all with or without '.gz')"
+            },
+            "contaminant_reference": {
+                "errorMessage": "Fasta file for decontamination, cannot contain spaces and must have extension '.fa', '.fasta', '.fna', '.fa.gz', '.fasta.gz or '.fna.gz'",
```

Comment:
that description is not informative I think..

Reply:
yeah.. let me re-work it